<html>
    <head>
<!-------------------------------------------------------------------------------------------------------------------------->
<!----------------------------------------------- Esto va en todas las paginas --------------------------------------------->
<!------------- Llama todo el css necesario y tiene la función de javascript para llamar al header y footer ---------------->
<!-------------------------------------------------------------------------------------------------------------------------->
        <title>BIOMOD 2019 Tec de Monterrey</title>
        <link href="https://biomod-tecmonterrey.github.io/BIOMOD2019/css/reset.css" rel="stylesheet"/>
        <link href="https://biomod-tecmonterrey.github.io/BIOMOD2019/css/style.css" rel="stylesheet"/>
        <meta charset="utf-8">
        <meta name="viewport" content="width=device-width,initial-scale=1.0">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
        <link href="https://biomod-tecmonterrey.github.io/BIOMOD2019/css/bootstrapmin.css" rel="stylesheet">
        <script src="https://code.jquery.com/jquery-3.4.1.min.js" type="text/javascript"></script>
        <script>
            $(function(){
                $("#div_header").load("https://biomod-tecmonterrey.github.io/BIOMOD2019/header.html");
                $("#div_footer").load("https://biomod-tecmonterrey.github.io/BIOMOD2019/footer.html");
            });
        </script>
<!-------------------------------------------------------------------------------------------------------------------------->
<!-------------------------------------------------------------------------------------------------------------------------->
<!-------------------------------------------------------------------------------------------------------------------------->
        <style>
            .articulo {
               font-size: 1vw;
               background: url(https://github.com/BIOMOD-TecMonterrey/BIOMOD2019/blob/master/images/otras/fondostrandsunidas.png?raw=true);
               background-size: contain;
               background-repeat-y: repeat;
               background-repeat-x: no-repeat;
               text-align: justify;
            }
            .masthead {
              background: url(http://2018.igem.org/wiki/images/2/2f/T--Tec-Monterrey--Header_Parts.jpg);
              background-size: cover;
              background-repeat: no-repeat;
              background-position-x: center;
              background-position-y: center;
            }
            .header-background {
              display: flex;
              z-index: 1;
              height: 83%;
              width: 99%;
              margin: 0;
              padding: 0;
              position: absolute;
              direction: rtl;
            }
            .header-background-left, .header-background-center, .header-background-center-right, .header-background-right {
              width: 18%;
            }
            .trail {
              width: 100%;
              height: auto;
              top: 5%;
            }
        
        </style>
<!-------------------------------------------------------------------------------------------------------------------------->
<!-------------------------------------------------------------------------------------------------------------------------->
<!-------------------------------------------------------------------------------------------------------------------------->
    </head>
    <body>
        <!-- Conn este div, y su id, javascript de head llama al header.html -->
        <div id="div_header" display="contents"></div>
        <!-- Este div no es utilizado por javascript, pero sirve para organizar la pagina -->
        <div id="page-content" display="contents">
            <!-- Header -->
            <header class="masthead" id="page-top">
                <div class= "design-title" id="design-title">DESIGN</div>
                
            </header>
            <div class="articulo">
                <section id="background" class="seccion-responsiva">
                    <div class="body-title">Background</div>
                    <br>
                        <div class ="body-subtitle">DNA Origami</div>
                            “DNA is the polymer that scientists dream about” [1]. Besides containing all human genetic information, its physical properties make it perfect to build nanoscale structures due to the following:  
                                1.       Watson-Crick base pairing complementarity
                                2.       Ladder of antiparallel strands
                                3.       Right-handed twist for B-DNA
                            These three features allowed the discovery of DNA nanotechnology; construction of two and three dimensional shapes using DNA as a building material. Throughout the years, distinct branches of this field have been developed including DNA Origami [2].
                            Building beautiful, complex and functional structures by folding a piece of paper is much more transcendental than the macroscale art we all know as origami. In 2006, Paul Rothemund discovered a way in which DNA, the molecule of life, could be adapted into the Japanese technique of origami.
                            The culmination of this discovery made scientists acknowledge that DNA Origami is one of the most powerful forms of DNA nanotechnology. The technique consists of folding a long strand of ssDNA (scaffold) using small oligonucleotides (staples) complementary to specific sections of the scaffold; achieving the construction of two and three dimensional structures [3] that can be potentially used in the areas of nanomedicine, physics, supramolecular chemistry and others. 
                        <div class="body-subtitle">Strand Displacement</div>
                            Strand displacement is a mechanism applied in the field of DNA Origami that allows DNA structures to be more dynamic and versatile. Also known as “toehold-mediated strand displacement”, the mechanism consists of replacing one strand in a double helix with another one, the resulting double helix will be more stable and favorable in terms of free energy than the initial one.
                        <div class="body-subtitle">DNA-RNA Hybrids </div>
                            In human biology, RNA plays a vital role as a molecule of life. Part of this is due to its structure and chemical properties. Yet, it is also because its complexity that molecular RNA de novo design and engineering is limited [4]. As a solution, folding RNA has been proposed as an adaptation of the DNA origami technique. 
                            DNA-RNA hybridization attempts to give DNA nanotechnology a new field of application and functionality. The folding process of the RNA follows the same principles of DNA origami [5]; the complementarity between two strands, in this case, RNA and DNA. And similarly, RNA-DNA origami relies on protocols already used for nucleic-acid self-assembly.
                            Distinct works have been developed in this area. There have been studies of the possible methodologies to hybridize these two molecules into complex non-functional nanostructures [4]. It has also been applied to construct RNA-DNA as siRNA delivery [6]. 
                        <div class="body-subtitle">mRNA Therapy </div>
                            <div class= "body-subsubtitle">what is mRNA?</div>
                                To answer this question, it is important to define what a protein is. A protein is a “workhorse” of the cell [7], they carry out vital functions of life; enzymes, hormones, antibodies, they are all proteins.
                                Proteins are expressed by genes; which are specific sections of DNA. The manufacture from DNA to proteins happens in two main steps: transcription and translation. In the first step, a DNA sequence is transcribed into messenger RNA (mRNA). In the second step, mRNA is translated into the amino acids that build up a protein [8]. Fundamentally, messenger RNA is a set of instructions to express any protein inside the human body, or more generally, inside any cell. 
                            <div class= "mRNA as a therapy">what is mRNA?</div>
                            <div class= "why?">what is mRNA?</div>
                            <ol type="1">
                                <li>Any protein can be encoded by mRNA
                                    <ul>
                                        <li>Enabling the development of prophylactic and therapeutic vaccines fighting diseases as diverse infections and cancer as well as protein replacement therapies.</li>
                                    </ul>
                                </li>
                                <li>Follows native biology [11].
                                    <ul>
                                        <li>In order to express any protein or protein complexes, only natural cell processes are followed.</li>
                                    </ul>
                                </li>
                                <li>mRNA does not interact with the genome [12], yet it harbors the exact elements required to express any encoded protein.
                                    <ul>
                                        <li>It does not enter the nucleus of a cell, therefore, detrimental integration with the genome is excluded.</li>
                                        <li>By being non-replicative and metabolically decaying within a few days, it is merely a transient carrier of information.</li>
                                    </ul>
                                </li>
                                <li>Comprises important pharmacologic characteristics.
                                    <ul>
                                        <li>The quantity supplied into the body can be controlled by doses.</li>
                                        <li>Strong and potent.</li>
                                        <li>Low cost manufacture [13].</li>
                                    </ul>
                                    
                                </li>
                            </ol>
                    

                </section>


                <section id="design" class="seccion-responsiva">
                    <div class="body-title">Design</div>
                    <div class="body-subtitle">The problem</div> 
                        Our project is based on the idea the mRNA needs to be stabilized inside the human body in order to be delivered as a drug. 
                        By stabilizing mRNA we believe that mRNA half-life and translation levels can be increased, leading to the overall production of significant levels of the encoded protein over a longer timeframe [14]. 
                        Naked mRNA is quickly degraded by extracellular RNases and is not internalized efficiently [13]. Therefore, a great variety of in vitro and in vivo transfection reagents have been developed that facilitate cellular uptake of mRNA. These reagents can be divided into three main categories: physical methods (including electroporation and sonication); viral-based approaches, and nonviral vectors ( such as lipid-based delivery systems) [15].
                        mRNA based therapies are not the only RNA-related treatments that are being developed worldwide; small interfering RNA therapies are used in order to block protein expression [14]. The small size of siRNA allow it to be more easily protected against degradation than mRNA. Because its size, molecular weight, stability, and tendency to form secondary structures, mRNA delivery presents a more challenging endeavor. 
                        Our solution is to build a structure that stabilizes the mRNA molecule inside the human body.
And what better way to do it than through a complementarity system? 
 
                    <div class="imagen-centrada-reducida">
                        <img src="http://2018.igem.org/wiki/images/8/85/T--Tec-Monterrey--RT_parts.png"> 
                    </div> 

                    <div class="body-subtitle">Wild Type Cas1 Protein: BBa_K2761001</div>
                        Gene sequence for the expression of Cas1 from Escherichia coli in its wild type form. This protein is one of the two Cas proteins responsible for new DNA spacer acquisition in the bacteria's CRISPR locus (array). This action is done by a protein complex hexamere made up of 4 Cas1 dimers and 2 Cas2 dimers. It has been shown that these proteins are commonly conserved across the major types of CRISPR-Cas systems. To properly use Cas1 protein it should be expressed along with Cas2 protein and a PAM sequence found in the msDNA in the system design for spacer acquisition. The Cas1 protein gene contained in this part expresses the protein without any tag for further identification.
                    <div class="imagen-centrada-reducida">
                        <img src="http://2018.igem.org/wiki/images/0/03/T--Tec-Monterrey--Cas1.png"> 
                    </div> 
                        Protein gene for the expression of Cas2 from Escherichia coli in its wild type form. The basic part includes Cas2 protein from Escherichia coli. This protein is one of the two Cas proteins responsible for new DNA spacer acquisition in the bacteria's CRISPR locus (array). This action is done by a protein complex hexamere made up of 4 Cas1 dimers and 2 Cas2 dimers. It has been shown that these proteins are commonly conserved across the major types of CRISPR-Cas systems. To properly use Cas2 protein it should be expressed along with Cas1 protein and a PAM sequence found in the msDNA in the system design for spacer acquisition. The Cas2 protein gene contained in this part expresses the protein without any tag for further identification
                    <div class="imagen-centrada-reducida">
                        <img src="http://2018.igem.org/wiki/images/4/4b/T--Tec-Monterrey--Cas2.png"> 
                    </div> 
                    <div class="body-subtitle">Cas1 Protein with Flag: BBa_K2761003</div>

                        Protein coding gene for the expression of Cas1 from Escherichia coli including a FLAG tag for its identification and purification. This protein is one of the two Cas proteins responsible for new DNA spacer acquisition in the bacteria's CRISPR locus. This action is done by a protein complex hexamere made up of 4 Cas1 proteins and 2 Cas2 proteins. It has been shown that these proteins are commonly conserved across the major types of CRISPR-Cas systems. To properly use Cas1 protein it should be expressed along with Cas2 protein and a PAM sequence found in the msDNA in the system design for spacer acquisition. The Cas1 protein gene contained in this part expresses the protein with a tag sequence (FLAG) for further identification and purification. 
                    <div class="imagen-centrada-reducida">
                        <img src="http://2018.igem.org/wiki/images/d/da/T--Tec-Monterrey--Cas1_flag.png"> 
                    </div> 
                    <div class="body-subtitle">Cas2 Protein with HA: BBa_K2761004</div>
                        Protein coding gene for the expression of Cas2 from Escherichia coli including a HA tag for its identification and purification. This protein is one of the two Cas proteins responsible for new DNA spacer acquisition in the bacteria's CRISPR locus. This action is done by a protein complex hexamere made up of 4 Cas1 proteins and 2 Cas2 proteins. It has been shown that these proteins are commonly conserved across the major types of CRISPR-Cas systems.To properly use Cas2 protein it should be expressed along with Cas1 protein and a PAM sequence found in the msDNA in the system design for spacer acquisition. The Cas2 protein gene contained in this part expresses the protein with a tag (HA) for further identification and purification.
                    <div class="imagen-centrada-reducida">
                        <img src="http://2018.igem.org/wiki/images/3/3b/T--Tec-Monterrey--Cas2_HA.png"> 
                    </div> 
                    <div class="body-subtitle">NsrR: BBa_K2761005</div>
                        This basic part consists of an optimized NsrR protein gene, which acts as a repressor of PyeaR promoter and is inhibited by the nitrate ion. E. coli natively produces a certain amount of NsrR. As such, the overexpression of NsrR can reduce any basal expression of GFP expressed by the PyeaR construct BBa_K381001 that may not be silenced by native NsrR. With a PyeaR reporter construct, the NsrR can be used to determine the presence of nitrate ion without any leaking.
                    <div class="imagen-centrada-reducida">
                        <img src="http://2018.igem.org/wiki/images/6/69/T--Tec-Monterrey--NsrR.png"> 
                    </div> 
                    <div class="body-subtitle">RBS for Cas1 and Cas2 proteins: BBa_K2761006</div>
                        RBS sequence used for the expression of Cas1 and Cas2 proteins. 
                    <div class="imagen-centrada-reducida">
                        <img src="http://2018.igem.org/wiki/images/6/6b/T--Tec-Monterrey--RBS_PARTS.png"> 
                    </div> 
                    <div class="body-subtitle">msr-msd Scaffold: BBa_K2761008</div>
                        Sequence for msr-msd transcript from EC86 retron, with specific PAM sequence for Cas1-Cas2 recognition.The msd-msr is transcribed as a single RNA sequence and interacts with the Retrotranscriptase (RT) EC86 protein to form an RNA-DNA chimera. The msr serves as a primer for the RT to start the retrotranscription, and the msd is the part of the transcript that is converted back into DNA. The msd is composed of a conserved part and a variable part. The conserved part is needed for retrotranscription and retron stability, while the variable part may be changed for any sequence of interest. In this part, the variable part of the msd contains an improved sequence that have high integration rates in the CRISPR array by adaptation proteins Cas1 and Cas2.
                </section>


                <section id="model" class="seccion-responsiva">
                    <div class="body-title">Model</div>
                    <div class="body-subtitle">Ne</div>
                       The composite parts BBa_K2761007 and BBa_K2761009 hold an IPTG-inducible promoter regulating the expression of each Cas1 and Cas2 proteins. The first part produces each protein with a FLAG and HA tags, respectively, the second one produces them as wild-type proteins. The expression of the proteins was characterized, quantifying them with a gradient of inductor concentration and under different induction times. These are new characterized parts for Silver requirements. The characterization of the part can be found in the Demonstrate segment. 
                    <br>
                    <div class="imagen-izquierda">
                        <img src="http://2018.igem.org/wiki/images/8/8c/T--Tec-Monterrey--Crispr_Construct.png">
                    </div> 
                    <br><br>
                    <div class="body-subtitle">Improved Part: RT & msd-msr: BBa_K2761010 </div>
                       The composite part BBa_K2761010 holds an IPTG-inducible promoter regulating the expression of a retrotranscriptase and the msr-msd transcript. The expressed RT comes with a histidine tag. The RT cassette for the new parts were improved from Part BBa_K1681000 by iGEM Team UIUC Illinois. However, the new parts are designed with a different msr-msd transcribed under a separate promoter cassette. The expression of the protein was characterized, quantifying it with a gradient of inductor concentration and under different induction times. This characterized part is an improvement of a previous part for Gold requirements. The characterization of this part can be found in the Demonstrate segment.  
                    <br>
                    <div class="imagen-izquierda">
                        <img src="http://2018.igem.org/wiki/images/3/37/T--Tec-Monterrey--Parts_RTconstruct.png" usemap="Mapaimagen1">
                    </div> 
                    <div class="body-subtitle">Improved Part:Nitrate inducible promoter: BBa_K2761011</div>
                      The composite part BBa_K2761011 contains the NsrR gen regulated by a constitutive promoter. This segment was ligated to BBa_K381001 which contains a Pyear promoter and a GFP reporter gen. Pyear is inhibit by the presence of NsrR which is natively produced by the bacteria. Under this condition no fluorescence is expected. Still is reported that under this condition, there is leaking activity that produces GFP. An over expression of the NsrR protein was proposed to overcome this problem. The presence of nitrate inhibits the NsrR protein from interfering with the promoter. This improvement intents to give the fluorescence response greater sensibility to nitrate ion. This characterized part is an improvement of a previous part for Gold requirements. The characterization of this part can be found in the Demonstrate segment.
                    <div class="imagen-centrada-reducida">
                        <img src="http://2018.igem.org/wiki/images/9/90/T--Tec-Monterrey--NsrR_construct.png"> 
                    </div> 
                    <br><br>
                </section>

                <section id="goals" class="seccion-responsiva">
                    <div class="body-title">Goals</div>
                    <br>
                       E-coding design a set of composite parts that enable the recording of stimuli from environment and culture conditions in the bacterial genome. The SCRIBE system together with the CAS1-CAS2 complex enables the integration of any designed sequence in the CRISPR-array. The project expects that the following collection of parts can be used in any type of bacteria for molecular recording of events. In this case, all experimentation was base in strain <i>.BL21</i> (DE3) Further experimentation in other types of strain is encouraged. 
                     <ol>
                     <li>
                         Cas1-Cas2 cassette with wild type proteins for integration in the bacteria CRISPR array
                    </li>
                     <li>
                         Cas1-Cas2 cassette with tag proteins, Flag and HA respectively, for rapid identification of protein production and activity of integration.
                     </li>
                     <li>
                         RT-His cassette for the production of a retrotranscriptase with a histidine tag. Cassette involves for the production of the desired sequence for integration of the genome and analog for the stimulus in measurement.
                     </li>
                     <li> 
                         Ns production. This cassette is used primarily to sense the presence of nitrates in the environment. 
                    </li>
                    </ol>
                </section>
            </div><!-- Termina articulo -->
        </div><!-- Termina page-content -->
        <!-- Conn este div, y su id, javascript de head llama al footer.html -->
        <div id="div_footer" display="contents"></div>
    </body>
</html>